server/middlewares/officeValidators.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable eol-last */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable consistent-return */
+/* eslint-disable indent */
+
+import express from 'express';
+import bodyParser from 'body-parser';
+import { Office } from '../helpers';
+
+
+const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+
+class OfficeValidators {
+    isOfficeFieldEmpty(req, res, next) {
+        const { officeName, officeType } = req.body;
+        const errors = {};
+        if (!officeName || !officeName.trim() || !officeType || !officeType.trim()) {
```

Comment:
Why do we have the first if condition that encloses others? is that necessary?

Reply:
I put it there as the main error catching block. The other if statements are to populate the error array if any of them exist.